### ISSUE TITLE
Sending many notifications via CSV through admin does not automatically use bulk queue

### ISSUE THREAD

**Issue by andrewleith** [cds-snc/notification-planning#749]
## Describe the bug 

When using admin, if a user chooses to send via CSV, the queue to be used should automatically be the bulk queue (`bulk-tasks`) when the number of rows in the CSV goes over `config.CSV_BULK_REDIRECT_THRESHOLD` (currently 200).

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
SEV-3

## Expected behavior

CSV uploads with more than `config.CSV_BULK_REDIRECT_THRESHOLD` rows should automatically be assigned to the bulk queue.

**check that this won't override if the template is set to Priority**  👉Why not? I think priority templates should be overriden, especially if someone sends a multi thousands rows CSV that will slow down all 2FA from a few seconds to many minutes. (Jimmy)

**Comment by yaelberger-commits** [comment]
Does this still matter given we default to bulk? @jimleroyer

**Comment by jimleroyer** [comment]
Yes if that's a bug. Normally, CSV uploads getting higher than a certain number of rows  should default to bulk, even if these are set to some other priorities. We could review the rules behind these and refine it at the same time.

**Comment by yaelberger-commits** [comment]
@jimleroyer Has this bug been fixed by some of your work on CSVs and bulk vs normal?

**Comment by jimleroyer** [comment]
@yaelberger-commits 

**tl;dr**: The queue is properly switched and used, but it's not reported correctly. You can close this bug as we have another one for [incorrect logging of the used queue](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/45).

Checking the queue that was used during a CSV upload, I confirm that the right queue is picked. On the other hand, it does not report properly in the logged column of the notification table in the database. I will assume that @andrewleith was using this to observe the used queue, which unfortunately, does not report it correctly. We have [a task to fix the reported queue](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/45) in the database (that I moved to the core board). I can chat with @andrewleith to share what I discovered and see if that matches his verifications.
